modules/bmo-editing-a-baremetalhost-resource.adoc

```diff
@@ -0,0 +1,46 @@
+// This module is included in the following assemblies: 
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="editing-a-baremetalhost-resource_{context}"]
+= Editing a BareMetalHost resource
+
+After deploying a cluster, you might need to edit a node's `BareMetalHost` resource. For example, if you deploy a cluster with the {ai-full} and you need to add baseboard management controller credentials, or you want to move a node from one cluster to another without deprovisioning it, you must edit the node's `BareMetalHost` resource. To edit a node's `BareMetalHost` resource without deprovisioning the node, use the following procedure.
```

Comment:
baseboard management controller > baseboard management controller (BMC)

Reply:
I see this acronym used in our docs.

Reply:
Editing a BMH's credentials and moving a BMH from one cluster to another are two vastly different scenarios. The first does not require the BMH to be detached. Editing BMC address, on the other hand, requires either the BMH to be in the Registering state, or to be detached.